src/anndata/_core/sparse_dataset.py

```diff
@@ -181,7 +181,11 @@ def _get_sliceXslice(self, row: slice, col: slice) -> ss.csr_matrix:
                     self._get_contiguous_compressed_slice(row), shape=out_shape
                 )
             return self._get_arrayXslice(np.arange(*row.indices(self.shape[0])), col)
-        return super()._get_sliceXslice(row, col)
+
+        # This is better than super()._get_sliceXslice,
+        # as that loads the complete .data and .indices arrays into memory
+        # TODO: Can probably still be more efficient
```

Comment:
I don't think this TODO is meaningful.  We're not going to be writing C code to do this, and I don't think it would be a huge improvement anyway.